pkg/webui/lib/errors/utils.js

```diff
@@ -124,12 +124,45 @@ export const getBackendErrorId = error => error.message.split(' ')[0]
  * @param {Object} error - The backend error object.
  * @returns {string} The default message.
  */
-export const getBackendErrorDefaultMessage = error =>
-  error.details[0].message_format || error.message.replace(/^.*\s/, '')
+export function getBackendErrorDefaultMessage(error) {
+  if (error.details[0].cause) {
+    return getBackendErrorCause(error.details[0].cause)
+  }
+  return error.details[0].message_format || error.message.replace(/^.*\s/, '')
+}
+
+/**
+ * Returns the error cause message of the error.
+ * @param {Object} cause - The backend error cause.
+ * @returns {string} The cause message.
+ */
+export function getBackendErrorCause(cause) {
```

Comment:
I think it would be better if the argument was always the root error object, otherwise, the usage might become confusing.